test/recipes/80-test_cms.t

```diff
@@ -222,13 +222,15 @@ my @smime_pkcs7_tests = (
       \&final_compare
     ],
 
-    [ "enveloped content test streaming S/MIME format, DES, 3 recipients, key only used",
+    [ "enveloped content test streaming S/MIME format, DES, 3 recipients, crt and pri files used",
       [ "{cmd1}", @defaultprov, "-encrypt", "-in", $smcont,
         "-stream", "-out", "{output}.cms",
         $smrsa1,
         catfile($smdir, "smrsa2.pem"),
-        catfile($smdir, "smrsa3.pem") ],
-      [ "{cmd2}", @defaultprov, "-decrypt", "-inkey", catfile($smdir, "smrsa3.pem"),
+        catfile($smdir, "smrsa3.crt") ],
+      [ "{cmd2}", @defaultprov, "-decrypt",
+	"-recip", catfile($smdir, "smrsa3.crt"),
+	"-inkey", catfile($smdir, "smrsa3.pri"),
```

Comment:
We do not use .pri for private keys. We sometimes use  .crt for certs so that is less problematic although there are no .crt files in smime-certs subdirectory yet. Please either rename both to smrsa3-cert.pem and smrsa3-key.pem or at least the key to smrsa3.key.

Reply:
done :-)